packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -323,24 +332,6 @@ describe('toggling dropdown', () => {
     });
   });
 
-  describe('empty list', () => {
-    it('vaadin-combo-box-overlay should not be attached when filteredItems is empty', () => {
-      comboBox.open();
-      expect(comboBox.opened).to.be.true;
-      expect(document.querySelector('vaadin-combo-box-overlay')).to.be.ok;
-      comboBox.close();
-      expect(comboBox.opened).to.be.false;
-      expect(document.querySelector('vaadin-combo-box-overlay')).not.to.be.ok;
```

Comment:
Removed this test because it's enough to check that `overlay.opened` is false. Teleporting is tested in overlay tests.